prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -53,7 +53,19 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - <<: *istio_container
+      # Lint requires a large amount of memory
+      # See https://github.com/istio/istio/issues/14888 before lowering requests
+      - image: gcr.io/istio-testing/istio-builder:v20190628-31457b43
+        # Docker in Docker
+        securityContext:
+          privileged: true
```

Comment:
We need this because it calls docker build?

Reply:
Is this because we call out to docker run somewhere in the script? We should refactor that to be pure binaries (or package it in the builder).

Reply:
This was a copy paste of the previous `istio_container` we were using. I don't think the linter specifically needs this, we just were using the same config for all our tests and I guess one of them needed it?

I don't think the lint uses docker